src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Extensions/MetadataAttributeTargetExtension.cs

```diff
@@ -79,7 +79,7 @@ public void WriteRazorSourceChecksumAttribute(CodeRenderingContext context, Razo
         context.CodeWriter.Write("[");
         context.CodeWriter.Write(SourceChecksumAttributeName);
         context.CodeWriter.Write("(@\"");
-        context.CodeWriter.Write(node.ChecksumAlgorithm);
+        context.CodeWriter.Write(node.ChecksumAlgorithm.ToString());
```

Comment:
Did you consider putting `.ToUpper()` here? Seems like it would make the baseline diff here significantly smaller.

Reply:
I don't think it would meaningfully make the diff smaller. All the checksums are still changing, so we don't even reduce the number of lines that have changed.